<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLoginComponent</class>
 <widget class="QFrame" name="CLoginComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login component</string>
  </property>
  <layout class="QVBoxLayout" name="hl_Timeout_2">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QFrame" name="fr_LoginDisconnected">
     <layout class="QVBoxLayout" name="vl_LoginDisconnected">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QGroupBox" name="gb_Network">
        <property name="title">
         <string>Network</string>
        </property>
        <layout class="QVBoxLayout" name="vl_Network">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="BlackGui::Components::CNetworkDetailsComponent" name="comp_NetworkDetails">
           <property name="frameShape">
            <enum>QFrame::Panel</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gb_PilotsDetails">
        <property name="title">
         <string>Pilot's info</string>
        </property>
        <layout class="QVBoxLayout" name="vl_PilotIngoGroupBox">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item alignment="Qt::AlignTop">
          <widget class="BlackGui::Editors::CPilotForm" name="form_Pilot">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>60</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gb_OwnAircraft">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="title">
         <string>Own aircraft</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="vl_OwnAircraft">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="BlackGui::Components::COwnAircraftComponent" name="comp_OwnAircraft">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="fr_LogoffConfirmationConnected">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="vl_LogoffConfirmationConnected">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QGroupBox" name="gb_MatchingLogConnected">
        <property name="title">
         <string>Matching log.</string>
        </property>
        <layout class="QVBoxLayout" name="vl_MatchingLogConnected">
         <item>
          <widget class="BlackGui::Components::CModelMatcherLogEnable" name="comp_MatchingLog">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignTop">
       <widget class="QGroupBox" name="gb_CurrentLoginConnected">
        <property name="title">
         <string>Current server and aircraft</string>
        </property>
        <layout class="QVBoxLayout" name="vl_CurrentServer">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="BlackGui::Editors::CServerForm" name="frp_CurrentServer">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="fr_CurrentLoginAddInfo">
           <layout class="QGridLayout" name="gl_AddInfo" columnstretch="0,2,1">
            <item row="2" column="0">
             <widget class="QLabel" name="lbl_LoginSince">
              <property name="text">
               <string>Login since:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="lbl_LoginAsAircraft">
              <property name="text">
               <string>Aircraft:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1" colspan="2">
             <widget class="QLineEdit" name="le_LoginSince">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="lbl_LoginCallsign">
              <property name="toolTip">
               <string>Callsign/home</string>
              </property>
              <property name="text">
               <string>CS/home:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="le_LoginCallsign">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLineEdit" name="le_LoginHomeBase">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1" colspan="2">
             <widget class="QLineEdit" name="le_LoginAsAircaft">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="fr_TimeoutConnected">
        <property name="visible">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="hl_Timeout">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QProgressBar" name="pb_LogoffTimeout">
           <property name="maximum">
            <number>10</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
           <property name="format">
            <string>%v</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tb_Timeout">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignBottom">
    <widget class="QFrame" name="fr_Buttons">
     <layout class="QGridLayout" name="gl_Buttons">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="1" column="1">
       <spacer name="hs_Buttons">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pb_Cancel">
        <property name="text">
         <string>cancel</string>
        </property>
        <property name="shortcut">
         <string>Esc</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pb_Ok">
        <property name="text">
         <string>ok</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlackGui::Editors::CFsdSetupForm</class>
   <extends>QFrame</extends>
   <header>blackgui/editors/fsdsetupform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CServerListSelector</class>
   <extends>QComboBox</extends>
   <header>blackgui/components/serverlistselector.h</header>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CNetworkDetailsComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/networkdetailscomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::COwnAircraftComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/ownaircraftcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::CLoginModeButtons</class>
   <extends>QFrame</extends>
   <header>blackgui/loginmodebuttons.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::CTickLabel</class>
   <extends>QLabel</extends>
   <header>blackgui/ticklabel.h</header>
  </customwidget>
  <customwidget>
   <class>BlackGui::Editors::CServerForm</class>
   <extends>QFrame</extends>
   <header>blackgui/editors/serverform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CModelMatcherLogEnable</class>
   <extends>QFrame</extends>
   <header>blackgui/components/modelmatcherlogenable.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CDbAircraftIcaoSelectorComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/dbaircrafticaoselectorcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CDbAirlineIcaoSelectorComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/dbairlineicaoselectorcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Editors::CPilotForm</class>
   <extends>QFrame</extends>
   <header>blackgui/editors/pilotform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comp_NetworkDetails</tabstop>
  <tabstop>form_Pilot</tabstop>
  <tabstop>frp_CurrentServer</tabstop>
  <tabstop>le_LoginCallsign</tabstop>
  <tabstop>le_LoginHomeBase</tabstop>
  <tabstop>le_LoginAsAircaft</tabstop>
  <tabstop>le_LoginSince</tabstop>
  <tabstop>tb_Timeout</tabstop>
  <tabstop>pb_Ok</tabstop>
  <tabstop>pb_Cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
